generatebundlefile/scripts/policy.json

```diff
@@ -0,0 +1,10 @@
+{
+        "default": [{"type": "reject"}],
+        "transports": {
+                "docker": {
+                        "857151390494.dkr.ecr.us-west-2.amazonaws.com": [{"type": "insecureAcceptAnything"}],
+                        "646717423341.dkr.ecr.us-west-2.amazonaws.com": [{"type": "insecureAcceptAnything"}],
+                        "public.ecr.aws": [{"type": "insecureAcceptAnything"}]
```

Comment:
`insecureAcceptAnything` huh? I admittedly don't even know what this is for, but I don't like the sounds of it.

Reply:
This is needed for skopeo, although I just copied this from eks-anywhere

https://github.com/aws/eks-anywhere/blob/main/release/scripts/policy.json

Reply:
Added an issue for this.